core/field_checkbox.ts

```diff
@@ -20,14 +20,16 @@ import {Field, FieldConfig, FieldValidator} from './field.js';
 import * as fieldRegistry from './field_registry.js';
 import type {Sentinel} from './utils/sentinel.js';
 
-export type FieldCheckboxValidator = FieldValidator<boolean>;
+type LegacyBool = 'TRUE'|'FALSE';
+type FieldBool = LegacyBool|boolean;
```

Comment:
Good point - `FieldBool` is probably confusing since it matches the naming of the field classes.

Instead of `StringOrBool`, what do you think about `CheckboxBool`?

Reply:
sgtm